src/ui/buttons.js

```diff
@@ -39,13 +46,17 @@ export const buttons = () => {
   });
 
   btnFinish.addEventListener("click", function () {
-    if (
-      window.confirm(
-        "Вы уверены, что хотите завешить тест?\nТекущий результат будет потерян!"
-      )
-    ) {
+    let message = "";
+    if (langSelect.value === "ru") {
+      message = langArr["confirm-window-message"]["ru"];
+    } else if (langSelect.value === "uz") {
+      message = langArr["confirm-window-message"]["uz"];
+    } else {
+      message = langArr["confirm-window-message"]["en"];
+    }
+    if ((confirmWindow = window.confirm(`${message}`))) {
```

Comment:
better would be to make it with the modal window, but let's proceed for now and create a tech debt ticket to improve it.